hc/api/tests/test_ensuretriggers.py

```diff
@@ -8,14 +8,19 @@
 
 
 class EnsureTriggersTestCase(TestCase):
+    """Performs tests to ensure database triggers have been activated. This is by:
+
+    * Instantiating a last ping as now().
+    * Assert that alert_after has been changed.
+    """
 
     def test_ensure_triggers(self):
         Command().handle()
 
         check = Check.objects.create()
         assert check.alert_after is None
         self.assertIsNone(check.alert_after, msg=None)
-
+        #Instantiate the last ping
```

Comment:
Acccording to pep8, youshould have a blanck space after `#`

Reply:
A new line at the end of the file is required, According to Pep8 standards.